### ISSUE TITLE
Blackhawk / Gift Card Mall / Simon / 5Back

### ISSUE THREAD

**Issue by nmock** [stevenmirabito/balance-check#4]
Any ideas how to proceed with captcha verification with the `blackhawk` provider? Seems that they are now using recaptcha verification via [Arkose Labs](https://www.arkoselabs.com), moving away from Google's recaptcha service.

**Comment by stevenmirabito** [comment]
Yes, this is supported by Anti-CAPTCHA. The implementation will have to be updated for the new CAPTCHA provider.

**Comment by ruanxuyi** [comment]
Got error `"(ThreadPoolExecutor-0_0) [CRITICAL] Unable to find reCAPTCHA"` for `blackhawk`. Likely that they have changed the class id `g-recaptcha`?

**Comment by stevenmirabito** [comment]
I have updated the Blackhawk provider with support for the new Arkose Labs FunCaptcha, however I keep getting "Invalid Captcha" responses from their site when I test the provider (both proxyless and with a proxy). I'm not sure what's wrong, and I don't have the time to invest in debugging this further right now, but feel free to troubleshoot and please submit a PR if you get it working.

**Comment by Markc1488** [comment]
Are people still working on this?

**Comment by j4jstyle** [comment]
For Arkose labs, you need to solve the captcha first, and inject the token in headers when doing request.
You still need to play with sessionid, uuid and other stuff to succeed "Enforcement" related matters, else you will keep getting Invalid Captcha the whole time.
